package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "jshint *.js stores/*.js middleware/*.js middleware/auth-utils/*.js example/*.js",
     "format": "semistandard",
     "test": "tape test/unit/*.js test/*.js",
-    "prepublish": "nsp check",
+    "nsp-check": "nsp check",
```

Comment:
This was a "hack" added to product repository when we migrated to PNC. "nsp check" downloads a lot of stuff from internet which is blocked by firewall in product build resulting in build failure. Changing "prepublish" to "nsp-check" (non-existing phase) basically turned off this check. 
On the other hand I think we want this check in community builds but this change turns it off.